### ISSUE TITLE
shift in byte mode should produce syntax error

### ISSUE THREAD

**Issue by arcanist123** [abaplint/abaplint#1989]
This is not a valid syntax that is not recognised by linter

``` abap 
  DATA lv_temp TYPE string.
  lv_temp = 'abc'.
  SHIFT lv_temp BY 1 PLACES LEFT in byte mode.
  WRITE lv_temp.
```

**Comment by larshp** [comment]
```abap
DATA lv_temp TYPE string.
SHIFT lv_temp BY 1 PLACES LEFT IN BYTE MODE.
```

**Comment by larshp** [comment]
@arcanist123 try again, 2.77.5

**Comment by larshp** [comment]
closing, no reply
